app/routes/missions/swift.mdx

```diff
@@ -276,6 +276,41 @@ https://swift.gsfc.nasa.gov/archive/
   </table>
 </div>
 
+**JSON-Serialized GCN Notice Types in GCN Kafka:**
+
+The GUANO system distributes alerts and localizations for sub-threshold detections of gamma-ray transients around the times of other triggers.
+These are bursts that did not result in any of the automatic GCN Notices described above, and is the result of computationally expensive offline analysis.
+
+These notices are published on the GCN Kafka topic **swift.bat.guano**.
+[Detailed Description and Examples](https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json)
+
+<div>
+  <table className="usa-table">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Contents</th>
+        <th>Latency</th>
+      </tr>
+    </thead>
+    <tr>
+      <th> Alert </th>
+      <th> Detection of a burst </th>
+      <th> 5 min - 5 hours </th>
+    </tr>
+    <tr>
+      <th> Localization </th>
+      <th> Arcminute position or HEALPix map </th>
+      <th> 4 hours - 8 hours </th>
+    </tr>
+    <tr>
+      <th> Retraction </th>
+      <th> Retraction of an alert or localization </th>
+      <th> 4 hours - 1 day </th>
+    </tr>
+  </table>
+</div>
```

Comment:
Did you try formatting this as a [Markdown table](https://www.markdownguide.org/extended-syntax/#tables)? Was there a problem with how that rendered it?

Reply:
I just copied the way the other table of notice types on this page was already formatted.
Not sure who wrote that one.

Reply:
Well, two wrongs don't make a right.

Reply:
Please reformat it as a Markdown table.

Reply:
done